felix/bpf/mock/map.go

```diff
@@ -165,6 +165,10 @@ func (m *Map) CopyDeltaFromOldMap() error {
 	return nil
 }
 
+func (m *Map) DeletePreviousVersion() error {
```

Comment:
did you need to export it if you do not use it outside the maps package?

Reply:
Oh great spot thanks @tomastigera - No, made deletePreviousVersion() private accordingly